i6_models/parts/conformer/feedforward.py

```diff
@@ -0,0 +1,45 @@
+from torch import nn
+
+
+class ConformerFeedForwardv1(nn.Module):
+    """
+    Conformer feedforward module
+    """
+
+    def __init__(self, input_dim, hidden_dim, dropout=0.1):
+        """
+        :param int input_dim: input dimension
+        :param int hidden_dim: hidden dimension (normally set to 4*input_dim as suggested by the paper)
+        :param float dropout: dropout probability
+        """
+        super().__init__()
+
+        self.layer_norm = nn.LayerNorm(normalized_shape=input_dim)
+
+        self.linear_layer_1 = nn.Linear(
```

Comment:
We earlier called this `linear_ff`. Why this name change? I think it would help to keep it compatible.

Reply:
`linear_ff_1` is still not the same as `linear_ff`.